massa-models/src/operation.rs

```diff
@@ -831,16 +788,11 @@ impl WrappedOperation {
             OperationType::Transaction { amount, .. } => *amount,
             OperationType::RollBuy { roll_count } => roll_price.saturating_mul_u64(*roll_count),
             OperationType::RollSell { .. } => Amount::zero(),
-            OperationType::ExecuteSC {
-                max_gas, gas_price, ..
-            } => gas_price.saturating_mul_u64(*max_gas),
-            OperationType::CallSC {
-                max_gas,
-                gas_price,
-                coins,
-                ..
-            } => gas_price
-                .saturating_mul_u64(*max_gas)
+            OperationType::ExecuteSC { max_gas, .. } => {
+                Amount::from_str(&max_gas.to_string()).unwrap_or(Amount::default())
```

Comment:
here we should return just the `coins` property of executesc

also, going through strings is inefficient

Reply:
It seems OperationType::ExecuteSC does not have `coins`, would returning zero be appropriate?

Reply:
true, let's return Amount::zero() like in the case of RollSell

Reply:
> continuing over at #3173

so this PR is abandoned and you continue on 3173 ?